Skip editor processing on empty cutting list

wsmirnow opened the pull request:
Skip processing recording, if it isn't trimmed or cutted. This will speedup processing in this case and save storage and energy.

Do we really want to set this value in each workflow? Maybe it make sense to change the default value of the editor WOH? What do you think about?

### Your pull request should…

* [ ] have a concise title
* [ ] [close an accompanying issue](https://docs.opencast.org/develop/developer/#participate/development-process/#automatically-closing-issues-when-a-pr-is-merged) if one exists
* [ ] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [ ] include migration scripts and documentation, if appropriate
* [ ] pass automated tests
* [ ] have a clean commit history
* [ ] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

Arnei commented:
Related: [Editor always creates a smil-file](https://github.com/opencast/opencast-editor/issues/1182)

gregorydlogan commented:
Per the meeting:
- Needs release note, push to 16.x

wsmirnow committed:
Skip processing recording if it isn't trimmed or cutted per default

This will speedup processing in this case and save storage and energy. The default value of the editor WOH property skip-if-not-trimmed is changed to true.

wsmirnow commented:
Rebased to r/16.x and changed the default value as discussed in the dev meeting.
